chore(rust): make `cargo publish --workspace` work

Boshen opened the pull request:
closes #3227

`rolldown_testing` is changed to depend on path because of cyclic dependency with `rolldown`.

https://crates.io/crates/rolldown is released on top of this branch.

Published crates: 

```
rolldown_std_utils
rolldown_utils
rolldown_error
rolldown_ecmascript
rolldown_sourcemap
string_wizard
rolldown_common
rolldown_debug_action
rolldown_debug
rolldown_ecmascript_utils
rolldown_fs
rolldown_resolver
rolldown_plugin
rolldown_plugin_chunk_import_map
rolldown_plugin_data_uri
rolldown_plugin_hmr
rolldown_plugin_oxc_runtime
rolldown_tracing
rolldown_watcher
rolldown_workspace
rolldown
rolldown_plugin_alias
rolldown_plugin_utils
rolldown_plugin_asset
rolldown_plugin_asset_import_meta_url
rolldown_plugin_build_import_analysis
rolldown_plugin_dynamic_import_vars
rolldown_plugin_esm_external_require
rolldown_plugin_import_glob
rolldown_plugin_isolated_declaration
rolldown_plugin_json
rolldown_plugin_load_fallback
rolldown_plugin_manifest
rolldown_plugin_module_preload_polyfill
rolldown_plugin_react_refresh_wrapper
rolldown_plugin_replace
rolldown_plugin_reporter
rolldown_plugin_transform
rolldown_plugin_vite_css
rolldown_plugin_vite_resolve
rolldown_plugin_wasm_fallback
rolldown_plugin_wasm_helper
rolldown_plugin_web_worker_post
rolldown_filter_analyzer
rolldown_plugin_vite_css_post
rolldown_plugin_vite_html
```

hyf0 commented on crates/rolldown_plugin_vite_css/Cargo.toml:
```diff
@@ -6,6 +6,7 @@ homepage.workspace = true
 license.workspace = true
 publish = true
```
```suggestion
publish = false
```

Boshen commented:
### Merge activity

* **Oct 18, 6:09 AM UTC**: The merge label 'graphite: merge' was detected. This PR will be added to the [Graphite merge queue](https://app.graphite.dev/merges?org=rolldown&repo=rolldown) once it meets the requirements.
* **Oct 18, 6:22 AM UTC**: `Boshen` added this pull request to the [Graphite merge queue](https://app.graphite.dev/merges?org=rolldown&repo=rolldown).
* **Oct 18, 6:38 AM UTC**: Merged by the [Graphite merge queue](https://app.graphite.dev/merges?org=rolldown&repo=rolldown).

hyf0 commented on crates/rolldown_plugin_vite_css_post/Cargo.toml:
```diff
@@ -6,6 +6,7 @@ homepage.workspace = true
 license.workspace = true
 publish = true
```
```suggestion
publish = false
```

hyf0 commented on crates/rolldown_plugin_vite_html/Cargo.toml:
```diff
@@ -6,6 +6,7 @@ homepage.workspace = true
 license.workspace = true
 publish = true
```
```suggestion
publish = false
```

hyf0 commented on crates/rolldown_plugin_vite_resolve/Cargo.toml:
```diff
@@ -4,6 +4,7 @@ version = "0.1.0"
 edition = "2021"
 license = "MIT"
 publish = true
```
```suggestion
publish = false
```

Boshen committed:
chore(rust): make `cargo publish --workspace` work (#6287)

closes #3227

`rolldown_testing` is changed to depend on path because of cyclic dependency with `rolldown`.

https://crates.io/crates/rolldown is released on top of this branch.

Published crates:

```
rolldown_std_utils
rolldown_utils
rolldown_error
rolldown_ecmascript
rolldown_sourcemap
string_wizard
rolldown_common
rolldown_debug_action
rolldown_debug
rolldown_ecmascript_utils
rolldown_fs
rolldown_resolver
rolldown_plugin
rolldown_plugin_chunk_import_map
rolldown_plugin_data_uri
rolldown_plugin_hmr
rolldown_plugin_oxc_runtime
rolldown_tracing
rolldown_watcher
rolldown_workspace
rolldown
rolldown_plugin_alias
rolldown_plugin_utils
rolldown_plugin_asset
rolldown_plugin_asset_import_meta_url
rolldown_plugin_build_import_analysis
rolldown_plugin_dynamic_import_vars
rolldown_plugin_esm_external_require
rolldown_plugin_import_glob
rolldown_plugin_isolated_declaration
rolldown_plugin_json
rolldown_plugin_load_fallback
rolldown_plugin_manifest
rolldown_plugin_module_preload_polyfill
rolldown_plugin_react_refresh_wrapper
rolldown_plugin_replace
rolldown_plugin_reporter
rolldown_plugin_transform
rolldown_plugin_vite_css
rolldown_plugin_vite_resolve
rolldown_plugin_wasm_fallback
rolldown_plugin_wasm_helper
rolldown_plugin_web_worker_post
rolldown_filter_analyzer
rolldown_plugin_vite_css_post
rolldown_plugin_vite_html
```
